build/build-sdk-images/tool/base/Dockerfile

```diff
@@ -21,7 +21,7 @@ FROM debian:buster
 
 RUN apt-get update && apt-get install -y \
   build-essential autoconf libtool git pkg-config curl \
-  automake libtool curl make g++ unzip moreutils \
+  automake libz-dev curl make cmake g++ unzip moreutils \
```

Comment:
Add zlib package because the following error happened.
```
   #7 477.7 src/core/lib/compression/message_compress.cc:25:10: fatal error: zlib.h: No such file or directory
   #7 477.7  #include <zlib.h>
   #7 477.7           ^~~~~~~~
   #7 477.7 compilation terminated.
   #7 477.7 make: *** [Makefile:810: /var/local/git/grpc/objs/opt/src/core/lib/compression/message_compress.o] Error 1
```